src/commands/test.js

```diff
@@ -54,3 +46,37 @@ export function test(yargs) {
     process.exit(1);
   }
 }
+
+function runNpmTestCommand() {
+  try {
+    exec('npm test', { stdio: 'inherit' });
+  } catch (err) {
+    Log.error('\nTests failed.');
+    process.exit(1);
+  }
+}
+
+export function test(yargs) {
+  Log.args(yargs.argv);
+
+  Log.info('Setting up plugin imports...\n');
+  loadPlugins();
+
+  Log.info('Setting up style imports...\n');
+  loadStyles();
+
+  const appRoot = path.resolve('.').split('.meteor')[0];
+  const { scripts } = require(path.join(appRoot, './package.json')) || {};
```

Comment:
There are a bunch of [file system utility functions](https://github.com/reactioncommerce/reaction-cli/blob/master/src/utils/fs.js) that I created specifically for reading/writing/checking files in the Reaction directory, so this could be simplified down to...

```js
import { getJSONFromFile } from '../utils';

const { scripts } = getJSONFromFile('./package.json');
``` 
[Source](https://github.com/reactioncommerce/reaction-cli/blob/master/src/utils/fs.js#L77-L89) for `getJSONFromFile`

Note that it's synchronous and already has try/catch handling with logging and process management, so it's a nice time saver.